.storybook/config.js

```diff
@@ -14,15 +14,19 @@ const Stories = [
     'HeadingFour',
     'HeadingFive',
     'Paragraph',
+    'Small',
```

Comment:
Small What?!?!!!?!? This naming scheme though lol

Reply:
makes sense, HTML small tag, like the replacement for `<small>`